cli/joy.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+'use strict';
+const fs = require('fs');
+const prog = require('caporal');
+const path = require('path');
+const pa11y = require('pa11y');
+const Docker = require('./plugins/docker');
+const Test = require('./plugins/test');
+const util = require('util');
+
+class Joy {
+  constructor() {
+    this.env = process.env;
+    this.prog = prog;
+    this.prog.version('1.0.0');
+  }
+
+  addCommand(commandObj) {
+    new commandObj(this);
+  }
+
+
+  /**
+   * Indicates whether the CWD is a Joy enabled project (e.g. is the parent to a .joy subdirectory)
+   * 
+   */
+  isJoy() {
+    try {
+      const stats = fs.statSync(path.join(process.cwd(), './.joy'));
+      return stats && stats.isDirectory();
+    }
+    catch (e) {
```

Comment:
I usually use the word `exception` for the parameter name, to avoid confusion with the use of `e` for `error`.  I also write the catch line on the same line as the closing bracket for the try block, like with if/else blocks.

Reply:
How about `ex` and `err`? Shorter to type.